eclair-core/src/main/resources/reference.conf

```
@@ -57,7 +57,7 @@ eclair {
     option_onion_messages = optional
     option_channel_type = optional
     option_payment_metadata = optional
-    trampoline_payment = disabled
+    trampoline_payment_legacy = disabled
```

I think this is clearer:
```suggestion
    trampoline_payment_prototype = disabled
```

---

Agreed, done.